vscode-extension/media/SectionOne.md

```diff
@@ -0,0 +1,12 @@
+## API Keys
+
+Some models require API keys in order to run them. If you want to run any of the models below, please create a ".env" file in the root of your project and add the following keys:
```

Comment:
Just noting we should keep this up-to-date with how @rossdanlm is implementing .env support

Reply:
this page is to be deprecated.